python/ray/dag/tests/experimental/test_accelerated_dag.py

```diff
@@ -1888,15 +1860,22 @@ def test_intra_process_channel_with_multi_readers(
 
 
 class TestLeafNode:
+    """
+    Leaf nodes are invalid because the exception thrown by the leaf
+    node will not be propagated to the driver.
```

Comment:
```suggestion
    Leaf nodes are not allowed right now because the exception thrown by the leaf
    node will not be propagated to the driver and silently ignored, which is undesired.
```